app/assets/stylesheets/error.scss

```diff
@@ -1,5 +1,6 @@
-@import "provider/typography";
-@import "provider/logo";
+@use 'provider/colors' as *;
```

Comment:
Unlike `@import`, `@use` namespaces the imported stuff like functions, mixins and variables, so you would need to access them as such: `typography.line-height-times(24)`, `colors.$border-color` etc.

Adding `as *` expoxes all of these in the current scope, so you can avoid adding the prefix.